design/baremetal-operator/disable-introspection-intereface.md

```diff
@@ -0,0 +1,96 @@
+# disable-inspection API
+
+## Status
+
+implementable
+
+## Summary
+
+A declarative API is proposed to disable inspection of a BareMetalHost.
+
+## Motivation
+
+Related to the work to enable booting of a
+[LiveImage](https://github.com/metal3-io/metal3-docs/pull/150),
+there is the requirement to optionally disable inspection on initial
+registration of a BareMetalHost (so that the live image can boot
+more quickly, ref user stories below).
+
+### Goals
+
+- A declarative API to disable inspection on BMH registration
+- The option to update status data with a status annotation when inspection is
+  performed by some means other than the regular BMH flow
```

Comment:
We can already do this. I think the goal is really to be able to do it at any arbitrary point in time, right?

Reply:
@dhellmann Yes, IIRC currently the status annotation is only consumed on the very first reconcile?  So we'll need the ability to update it after provisioning in this case.

Reply:
Right, so I think the phrasing of the goal should reflect that.